app/views/logs/index.html.erb

```diff
@@ -24,7 +24,7 @@
           <div class="row">
             <td class="col-3"><%= log.title.upcase %></td>
             <td class="col-2"><%= log.model %></td>
-            <td class="col-2"><%= log.service_id %></td>
+            <td class="col-2"><%= log.service_ids.to_sentence %></td>
```

Comment:
if the association we made is correct, we should be accessing this through the availed_services association
log.availed_services

Reply:
![image](https://user-images.githubusercontent.com/100547162/183725275-7b043f90-1b40-4ba5-827a-6c5e3770eb1d.png)

Reply:
![image](https://user-images.githubusercontent.com/100547162/183725349-0cdb73f6-6453-4367-8d94-99f9030ce08a.png)

Reply:
seems like this is the output if availed_services is used. Do you reckon there's a problem somewhere along the association?

Reply:
as discussed